contracts/NonFungibleToken.cdc

```diff
@@ -71,6 +71,24 @@ pub contract interface NonFungibleToken {
     pub resource interface INFT {
         // The unique ID that each NFT has
         pub let id: UInt64
+
+        /// Function that returns all the Metadata Views implemented by a Non Fungible Token
```

Comment:
What is the reason for declaring these metadata function in `INFT`? 
Shouldn't the functions in `MetadataViews.Resolver` gain default implementations?

Reply:
because we want to upgrade the NFT standard without breaking every NFT contract. If we require every NFT contract to upgrade to use `Resolver`, then they'll all break

Reply:
I see. Basically the functions are only here to keep backwards-compatibility? 

Maybe add a comment explaining this. New readers of the contract might wonder and have the same question.

Reply:
Added a comment. 👍

Reply:
Thank you @turbolent. I have been dealing with the same issue on the FT repo and the best I can say now is that I guess I haven't interiorised yet default function implementations.